scripts/olm-create-bundle.sh

```diff
@@ -38,38 +37,33 @@ Usage:
-                                            Default: {SERVICE_CONTROLLER_SOURCE_PATH}/olm
-  AWS_SERVICE_DOCKER_IMG:                   Specify the docker image for the AWS service.
-                                            This should include the registry, namespace,
-                                            image name, and tag. Takes precedence over
-                                            SERVICE_CONTROLLER_CONTAINER_REPOSITORY
-  SERVICE_CONTROLLER_SOURCE_PATH:           Path to the service controller source code
-                                            repository.
-                                            Default: ../{SERVICE}-controller
-  SERVICE_CONTROLLER_CONTAINER_REPOSITORY   The container repository where the controller exists.
-                                            This should include the registry, namespace, and
-                                            image name.
-                                            Default: $DEFAULT_SERVICE_CONTROLLER_CONTAINER_REPOSITORY
-  TMP_DIR                                   Directory where kustomize assets will be temporarily
-                                            copied before they are modified and passed to bundle
-                                            generation logic.
-                                            Default: $BUILD_DIR/tmp-olm-{RANDOMSTRING}
-  PRESERVE:                                 Preserves modified kustomize configs for
-                                            inspection. (<true|false>)
-                                            Default: false
-  BUNDLE_GENERATE_EXTRA_ARGS                Extra arguments to pass into the command
-                                            'operator-sdk generate bundle'.
+  BUNDLE_DEFAULT_CHANNEL            The default channel to publish the OLM bundle to.
+                                    Default: $DEFAULT_BUNDLE_CHANNEL
+  BUNDLE_VERSION                    The semantic version of the bundle should it need
+                                    to differ from the version of the controller which is
+                                    passed into the script. Optional. If unset, this
+                                    value will match that passed in by the user as the
+                                    <version> parameter (i.e. the controller version)
+  BUNDLE_CHANNELS                   A comma-separated list of channels the bundle belongs
+                                    to (e.g. \"alpha,beta\").
+                                    Default: $DEFAULT_BUNDLE_CHANNEL
+  BUNDLE_OUTPUT_PATH:               Specify a path for the OLM bundle to output to.
+                                    Default: {SERVICE_CONTROLLER_SOURCE_PATH}/olm
+  ACK_GENERATE_IMAGE_REPOSITORY:    Specify a Docker image repository to use
+                                    for release artifacts
+                                    Default: public.ecr.aws/aws-controllers-k8s/{SERVICE}-controller
```

Comment:
Did you want to make the default for the OLM bundle creation script something that is more operatorhub.io-centric?

Reply:
@jaypipes I'm not sure I understand the question. This is where the controller's container images come from, so as long as they are in a a public repo (private is possible too, but need pull secrets configured), it doesn't matter if the image repo is `ecr`, `quay`, `docker`, `gcr`, etc. I don't think the ACK project is going to start hosting images in two places at this point, right?

Reply:
@acornett21 I am a dummy. You are absolutely correct. I was stupidly thinking this was the URI for sending the OLM bundles to.